CONTRIBUTING.md

```diff
@@ -8,6 +8,48 @@ When you submit a pull request, a CLA bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
+# Types of Contributions
+
+## Submit a pull request that aligns with our goals and priorities
+The STL repo has many filed issues that track work to be completed.
+If you're unsure of where to start, you may want to start by:
+ * Looking for pinned issues
+ * Checking issues under the labels `high priority` and `help wanted`
+ * New contributors may also look into issues labeled `good first issue`
+
+## Review a pull request
+
+We welcome code reviews from contributors! Reviews from other contributors can often accelerate the reviewing process
+by helping a PR reach a more finished state before maintainers review the changes.  As a result, such a PR may require
+fewer maintainer review iterations before reaching a "Ready to Merge" state!
+To gain insight into our Code Review process, you may want to view:
+ * Ongoing reviews in [Pull Requests List](https://github.com/microsoft/STL/pulls)

```

Comment:
```suggestion
 * pull requests which are [undergoing review][review:changes-requested],
```

(with a link block):

```md
[review:changes-requested]: /microsoft/STL/pulls?q=is%3Apr+is%3Aopen+review%3Achanges-requested
```

Reply:
I'm not really sure if it should be changes requested state. Maybe review requested state?